class.acpt.php

```diff
@@ -12,11 +12,8 @@ public function __construct() {
 			in_array( 'advanced-custom-fields/acf.php', $active_plugins )
 		);
 
-		if ( ! $acf_activated ) {
-			if ( is_admin() ) {
-				add_action( 'admin_notices', array( $this, 'admin_notice_acf_not_activated' ) );
-			}
-
+		if ( ! $acf_activated && is_admin() ) {
+			add_action( 'admin_notices', array( $this, 'admin_notice_acf_not_activated' ) );
 			return;
 		}
```

Comment:
The return is on the outside of the conditional. This would only halt the plugin load if you were in admin.